pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiActivityInstance.java

```diff
@@ -45,7 +45,7 @@ default String generateUniqueGuid() {
 
     @SqlQuery("select activity_instance_id from activity_instance where "
             + "activity_instance_guid = :activityInstanceGuid")
-    long getActivityInstanceId(String activityInstanceGuid);
+    long getActivityInstanceId(@Bind("activityInstanceGuid") String activityInstanceGuid);
 
```

Comment:
Trying to get past problems in my IDE. Once in a while my IntelliJ compiler simply will not apply it properly. Look forward to day when we can get rid of -parameter flag for real. A small contribution in that direction.